docker-compose.yaml.example

```diff
@@ -0,0 +1,28 @@
+version: "3.3"
```

Comment:
Needs at least version 3.7

Reply:
I'm not at my computer at the moment but I think you should be able to push to this branch?